.travis.yml

```diff
@@ -5,5 +5,6 @@ python:
 install:
         - pip install -r requirements.txt
 script:
+        - python -m unittest discover pydatastructs
```

Comment:
Not needed. The current framework of `pydatastructs` uses pytest for testing. Please remove this